src/styles/blocks/stars.css

```diff
@@ -0,0 +1,32 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: block;
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--0 :nth-child(-n + 5),
+.stars--1 :nth-child(n + 2),
+.stars--2 :nth-child(n + 3),
+.stars--3 :nth-child(n + 4),
+.stars--4 :nth-child(n + 5) {
+  background-image: url(../../images/star.svg);
+}
```

Comment:
```suggestion
.stars__star {
  display: block;
  background-repeat: no-repeat;
  background-position: center;
  background-image: url(../../images/star.svg);
  width: 16px;
  height: 16px;
  margin-right: 4px;
}

.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
  background-image: url(../../images/star-active.svg);
}
```

Reply:
set `star-img` for all stars, then just rewrite to active some of them

Reply:
Thank you for your comments! I've corrected my code. Could you please check it again?